canonical-websites/snapcraft.io
# Small fixes for blueprints

# Summary

- Rename blog blueprint to be more readable (`blog_page` -> `blog`)
- Rename store blueprint to be more readable (`store_page` -> `store`)
- Remove useless promoted view because it wasn't linked anywhere
- use `url_for` for redirects in the store view
- Make `url_for` more readable, instead of writing `blueprint.function` -> `.function` (only if this redirect is in the blueprint)
- Remove unsued endpoint (`/status` is now `/_status` provided by talisker)

# QA

- `./run`
- Visit
- http://127.0.0.1:8004/
- http://127.0.0.1:8004/store
- http://127.0.0.1:8004/discover
- http://127.0.0.1:8004/search
- http://127.0.0.1:8004/toto
- http://127.0.0.1:8004/blog
- Those urls should work as usual
